examples/image_classification/Image_Classification_using_PyTorch_and_🔭_Galileo.ipynb

```diff
@@ -158,10 +158,10 @@
     "    assert label_col in col_names, f\"{label_col} is not an existing column\"\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #57.      data['train'] = data['train'].filter(lambda example: example[label_col] == 0).select(range(100))
</pre><p> </p><p>Was referring to the comment above this line.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rungalileo/examples/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>